Changes

```diff
@@ -1,5 +1,18 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v5.0.2 2020-05-22
+
+ [Fixes]
+ - Some zones create too large result to fit into the zonemaster
+   database which made the Test Agent crasch (only MySQL or
+   MariaDB, not PostgreSQL, database was affected). This version
+   increases the database size (#616, #617).
+ - Use instructions on Zonemaster-Backend main README for
+   upgrading the zonemaster database.
```

Comment:
"Use" -> "Place"?

Reply:
Please clarify?

Reply:
I don't think the current wording is very clear, so I tried to guess its meaning.

Reply:
I've tried to improve. Else propose something.